v2/emailpassword/advanced-customizations/apis-override/usage.mdx

```diff
@@ -9,7 +9,9 @@ import TabItem from '@theme/TabItem';
 
 # How to use
 
-## Use the override config:
+## Method 1: Use the override config:
```

Comment:
You have to use Override config for both methods.. Change heading to "Method 1: Pre / Post API logic change"